CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * [CHANGE] Alertmanager: Local file disclosure vulnerability in OpsGenie configuration has been fixed. #5045
 * [CHANGE] Rename oltp_endpoint to otlp_endpoint to match opentelemetry spec and lib name. #5067
 * [CHANGE] Distributor/Ingester: Log warn level on push requests when they have status code 4xx. Do not log if status is 429. #5103
+* [CHANGE] Alertmanager: Route web-ui requests to the alertmanager distributor when sharding is enabled. #5120
```

Comment:
It's a more a bugfix than a change.

Reply:
Good point, I have updated that to a bugfix